linera-service/src/storage.rs

```diff
@@ -400,64 +429,64 @@ pub async fn full_initialize_storage(
     genesis_config: &GenesisConfig,
 ) -> Result<(), anyhow::Error> {
     match config {
-        StoreConfig::Memory(_) => {
+        StoreConfig::Memory(_, _) => {
             bail!("The initialization should not be called for memory");
         }
         #[cfg(feature = "rocksdb")]
-        StoreConfig::RocksDb(config) => {
+        StoreConfig::RocksDb(config, namespace) => {
             let wasm_runtime = None;
-            let mut storage = RocksDbStorage::initialize(config, wasm_runtime).await?;
+            let mut storage = RocksDbStorage::initialize(config, &namespace, wasm_runtime).await?;
             genesis_config.initialize_storage(&mut storage).await
```

Comment:
Here, whenever the DB namespace already existed, I would expect that `initialize_storage` will panic with an assert failure in `linera_storage::Storage::create_chain` (`"Attempting to create a chain twice"`).

Reply:
This behavior can be changed in future PRs.
I did not want to change the behavior of the code.